packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts

```diff
@@ -69,9 +70,21 @@ export default ({ api }: { api: APIType }) => {
     }
   }
 
+  const terminateCard = function* (action: ReturnType<typeof A.terminateCard>) {
+    try {
+      const { payload } = action
+      yield call(api.terminateDC, payload)
+
+      yield call(getCards)
+    } catch (e) {
+      console.error('Failed to terminate card', errorHandler(e))
+    }
```

Comment:
this feels like it needs to be a Remote.  how are you going to inform the user that we failed to terminate the card?

Reply:
We Are still discussing how to show that to the user with Cody. Also, this is for MVP, in V1 designs this feature will be shown in a different way. So I prefer not to spend much time on this